tests/test_version.py

```
@@ -0,0 +1,47 @@
+import os
+
+class TestVersionFlag(unittest.TestCase):
+    """Test the --version flag functionality"""
+    
+    def setUp(self):
+        """Set up test environment"""
+        self.script_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'મુખ્ય.py')
+    
+    def test_version_short_flag(self):
+        """Test -v flag"""
+        result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)